tests/topotests/ldp-vpls-topo1/test_ldp_vpls_topo1.py

```diff
@@ -257,6 +257,7 @@ def test_ldp_pseudowires_after_link_down():
     # Shut down r1-r2 link */
     tgen = get_topogen()
     tgen.gears['r1'].peer_link_enable('r1-eth1', False)
+    sleep(5)
```

Comment:
Is this `sleep` necessary? `router_compare_json_output` already does `run_and_expect`.

If you really want to go that way I suggest using `topotest.sleep` (preferably with a message) so we know we are waiting there.

Reply:
> Is this `sleep` necessary? `router_compare_json_output` already does `run_and_expect`.

The idea of this `sleep` is to give zebra some time to detect the link down failure and react accordingly. This is necessary because the output of "show l2vpn atom vc" is the same before and after the pseudowire reconvergence (it doesn't show the pseudowire resolved next-hop router).

> If you really want to go that way I suggest using `topotest.sleep` (preferably with a message) so we know we are waiting there.

That's a good suggestion, will do that :)